CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## [0.1.0](https://github.com/chainsafe/as-chacha20poly1305) (2022-10-24)
```

Comment:
Is the changelog necessary? Why not just rely on releases page?

Reply:
I think its autopopulated by the google release-bot?

Reply:
will remove it 👍